src/mlpack/tests/init_rules_test.cpp

```diff
@@ -197,6 +198,103 @@ BOOST_AUTO_TEST_CASE(GaussianInitTest)
   BOOST_REQUIRE_EQUAL(weights3d.n_slices, slices);
 }
 
+/**
+ * Simple test of the VarianceScalingNormalInit class.
+ */
+BOOST_AUTO_TEST_CASE(VarianceScalingNormalInitTest)
+{
+  const size_t rows = 7;
+  const size_t cols = 8;
+  const size_t slices = 2;
+
+  arma::mat weights;
+  arma::cube weights3d;
+
+  VarianceScalingNormalInit<> vsn;
+
+  vsn.Initialize(weights, rows, cols);
+  vsn.Initialize(weights3d, rows, cols, slices);
+
+
+  // Check that size of initialized weights is equal to the input weights size
+
+  BOOST_REQUIRE_EQUAL(weights.n_rows, rows);
+  BOOST_REQUIRE_EQUAL(weights.n_cols, cols);
+
+  BOOST_REQUIRE_EQUAL(weights3d.n_rows, rows);
+  BOOST_REQUIRE_EQUAL(weights3d.n_cols, cols);
+  BOOST_REQUIRE_EQUAL(weights3d.n_slices, slices);
+
+  // Check that the initialized values lie in the required range
+
+  double variance = (1.0 / ( ( rows + cols ) / 2.0 ) );
+  double stddev = sqrt(variance);
+
+  for (size_t i = 0; i < rows; i++)
+    for (size_t j = 0; j < cols; j++)
+    {
+      BOOST_REQUIRE_GE(weights.at(i, j), - 3*stddev);
+  	  BOOST_REQUIRE_LE(weights.at(i, j), 3*stddev);
+  	}
+
+  for (size_t k = 0; k < slices; k++)
+    for (size_t i = 0; i < rows; i++)
+      for (size_t j = 0; j < cols; j++)
+      {
+        BOOST_REQUIRE_GE(weights3d.slice(k).at(i, j), - 3*stddev);
+    	BOOST_REQUIRE_LE(weights3d.slice(k).at(i, j), 3*stddev);
+      }
```

Comment:
Some of the spacing and style does not seem to be right, especially in this section.  Could you go through the style guide and fix it please?  We normally have an automatic style checker but it is offline right now (hopefully it will be back online soon).

https://github.com/mlpack/mlpack/wiki/DesignGuidelines#style-guidelines

Reply:
I will fix the style issues asap